src/routes/_components/status/StatusMediaAttachments.html

```diff
@@ -233,7 +236,21 @@
       sensitiveMediaIconClass: ({ canUseBlurhash }) => classname(
         'status-sensitive-media-svg',
         canUseBlurhash && 'status-sensitive-media-svg-transparent'
-      )
+      ),
+      nCols:
+        ({ mediaAttachments, $largeInlineMedia }) => {
+          return (!$largeInlineMedia && mediaAttachments.length > 1) ? 2 : 1
+        },
```

Comment:
Is this `nCols` unused?

Reply:
It's OK, I can remove it in another PR :)

Reply:
Yes we can remove it - I must admit I did just copy the code and asumed that twoCols depended on nCols...